php-nginx/build-scripts/build_php56.sh

```diff
@@ -128,9 +121,10 @@ mkdir -p ${PHP56_DIR}/lib/conf.d
 
 # Install shared extensions
 ${PHP56_DIR}/bin/pecl install memcache
```

Comment:
I think we should remove it before this goes beta, as it may be confused with our (forthcoming) Memcached support.